components/butterfly/Cargo.toml

```diff
@@ -21,8 +21,10 @@ log = "*"
 lazy_static = "*"
 prometheus = "*"
 parking_lot = "*"
-prost = "*"
-prost-derive = "*"
+# Must use fork due to this issue https://github.com/danburkert/prost/issues/242
```

Comment:
It'd be a good idea to create an issue to track this fork situation.

Reply:
Added issue [here](https://github.com/habitat-sh/habitat/issues/7270)